nixos/modules/services/web-apps/plausible.nix

```diff
@@ -245,9 +245,11 @@ in {
             ${lib.optionalString (cfg.mail.smtp.passwordFile != null)
               ''export SMTP_USER_PWD="$(< $CREDENTIALS_DIRECTORY/SMTP_USER_PWD )"''}
 
+            printenv
```

Comment:
debug ?

Reply:
@kirillrdy good catch ! Yes it was for debugging forgot to remove it in my haste. Corrected and removed now.